src/main/java/frc/trigon/robot/subsystems/ledstrip/LEDStrip.java

```diff
@@ -0,0 +1,146 @@
+package frc.trigon.robot.subsystems.ledstrip;
+
+import com.ctre.phoenix.led.*;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class LEDStrip extends SubsystemBase {
+    private static final CANdle CANDLE = LEDStripConstants.CANDLE;
+    private static int lastCreatedLEDStrip = 0;
+    private final int slotID;
+    private final int offset, numLEDs;
+
+    public LEDStrip(int offset, int numLEDs) {
```

Comment:
give numLEDs a better name 
The fact that they gave it a bad name doesn't mean you need to 
Also, give the constructor a javadoc